widgetry/src/style/buttons.rs

```diff
@@ -2,6 +2,7 @@ use super::ButtonStyle;
 use crate::{
     include_labeled_bytes, ButtonBuilder, ControlState, EventCtx, ScreenDims, Style, Widget,
 };
+use geom::CornerRadii;
```

Comment:
Nit: Separate paragraph above the other imports

Reply:
derp. fixed